Gemfile

```diff
@@ -135,7 +135,7 @@ group :development, :test do
   gem 'rails-controller-testing'
   gem 'puma', '~> 5.6', '>= 5.6.8'
   gem 'railroady', '~> 1.5.2'
-  gem 'rspec-rails', '~> 5.1.2'
+  gem 'rspec-rails', '5.0.3'
```

Comment:
is there any issue that this was downgraded?

Reply:
good catch, probably not. But I had `gem 'rspec-rails', '~> 5.1.2'` in other pr when migrating to rails 6.1. I will update this branch too